SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialGameInstance.h

```diff
@@ -53,11 +54,17 @@ class SPATIALGDK_API USpatialGameInstance : public UGameInstance
 
 	void HandleOnConnected();
 	void HandleOnConnectionFailed(const FString& Reason);
+	void HandleOnPlayerSpawnFailed(const FString& Reason);
 
 	// Invoked when this worker has successfully connected to SpatialOS
+	UPROPERTY(BlueprintAssignable)
 	FOnConnectedEvent OnConnected;
```

Comment:
Raised by Miron in the other PR: we may want to change the names to `OnSpatialConnection` etc to be very clear that these will only be triggered when running on Spatial.